src/vis/image.js

```diff
@@ -85,6 +85,8 @@
 
       this.no_cdn = options.no_cdn;
 
+      this.auth_tokens = options.auth_tokens;
```

Comment:
@alonsogarciapablo do you think we should guard against null here?

```js
this.auth_tokens = options.auth_tokens || [];
```

Maybe this will break when no auth_tokens are passed in, and later down the line some array operation is performed on null.